apps/ejabberd/src/ejabberd.app.src

```diff
@@ -2,7 +2,7 @@
 
 {application, ejabberd,
  [{description, "ejabberd"},
-  {vsn, {cmd, "echo 2.1.8+mim-`git describe --always --tags`"}},
+  {vsn, {cmd, "echo 2.1.8-mim-`git describe --always --tags`"}},
```

Comment:
Why `-` is better than `+`. As far as I remember we introduced `+` to separate ejabberd base version from mongooseim version. I don't mind `-` but just wanted to know why we'd like to introduce this change.